src/alertsmanagement/azext_alertsmanagement/__init__.py

```diff
@@ -0,0 +1,32 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
is it necessary to put whole alertsmanagement folder under monitor folder? concern is the user need to install extra extension when using it under az monitor. monitor is in azure cli core right? @MyronFanQiu what's your thoughts

Reply:
offline synced, it's ok to keep it as extension